app/javascript/Components/ta_table.jsx

```diff
@@ -59,87 +113,7 @@ class TATable extends React.Component {
   render() {
     const {data} = this.state;
     return (
-      <ReactTable
-        data={data}
-        columns={[
-          {
-            Header: I18n.t("activerecord.attributes.user.user_name"),
-            accessor: "user_name",
-          },
-          {
-            Header: I18n.t("activerecord.attributes.user.first_name"),
-            accessor: "first_name",
-          },
-          {
-            Header: I18n.t("activerecord.attributes.user.last_name"),
-            accessor: "last_name",
-          },
-          {
-            Header: I18n.t("activerecord.attributes.user.email"),
-            accessor: "email",
-          },
-          {
-            Header: I18n.t("roles.active") + "?",
-            accessor: "hidden",
-            Cell: ({value}) => (value ? I18n.t("roles.inactive") : I18n.t("roles.active")),
-            filterMethod: (filter, row) => {
-              if (filter.value === "all") {
-                return true;
-              } else {
-                return (
-                  (filter.value === "active" && !row[filter.id]) ||
-                  (filter.value === "inactive" && row[filter.id])
-                );
-              }
-            },
-            Filter: selectFilter,
-            filterOptions: [
-              {
-                value: "active",
-                text: `${I18n.t("roles.active")} (${this.state.counts.active})`,
-              },
-              {
-                value: "inactive",
-                text: `${I18n.t("roles.inactive")} (${this.state.counts.inactive})`,
-              },
-            ],
-            filterAllOptionText: `${I18n.t("all")} (${this.state.counts.all})`,
-          },
-          {
-            Header: I18n.t("actions"),
-            accessor: "id",
-            Cell: data => (
-              <>
-                <span>
-                  <a
-                    href={Routes.edit_course_ta_path(this.props.course_id, data.value)}
-                    aria-label={I18n.t("edit")}
-                    title={I18n.t("edit")}
-                  >
-                    <FontAwesomeIcon icon={faPencil} />
-                  </a>
-                </span>
-                &nbsp;|&nbsp;
-                <span>
-                  <a
-                    href="#"
-                    onClick={() => this.removeTA(data.value)}
-                    aria-label={I18n.t("remove")}
-                    title={I18n.t("remove")}
-                  >
-                    <FontAwesomeIcon icon={faTrashCan} />
-                  </a>
-                </span>
-              </>
-            ),
-            filterable: false,
-            sortable: false,
-          },
-        ]}
-        filterable
-        loading={this.state.loading}
-        noDataText={I18n.t("tas.empty_table")}
-      />
+      <Table data={this.state.data} columns={this.columns} noDataText={I18n.t("tas.empty_table")} />
```

Comment:
Note that the `data` variable is defined immediately above this, so you can either delete that variable or just pass in `data` here.